docs/modules/ROOT/pages/data-migration.adoc

```diff
@@ -1,48 +1,34 @@
-= Migrating Data to Another Cluster
-
-The goal of this document is to describe how to migrate data from one cluster to another.
+= Migrate Data to another Cluster
+:description: In this tutorial, you'll use xref:wan-replication.adoc[WAN Replication] to migrate data from one cluster to another.
```

Comment:
You've been using `{description}` among other tutorials to display the metadata description. It's missing here - is it an overlook?